labeled_data/technology/database/document.yml

```diff
@@ -97,7 +100,7 @@ data:
     - 18351848 # repo:typicode/lowdb
     - 25364889 # repo:unnamed38/fueldb
     - 60377070 # repo:vespa-engine/vespa
+    - 735981 # repo:xapian/xapian
     - 125824259 # repo:xtdb/xtdb
     - 456549280 # repo:ydb-platform/ydb
     - 47479424 # repo:zerodb/zerodb
-    - 522606931 # repo:Anna-Team/AnnaDB
```

Comment:
I'm not sure why this repo deleted, is this right ?
I found this [repo](https://github.com/Anna-Team/AnnaDB)  is still alive.